### DISCUSSION TITLE
Formatter on exportRow fails if the function depends on rowData

### DISCUSSION THREAD

**Discussion by acemir** [IgniteUI/igniteui-angular#13127] [Ideas]
The exportRow's formatting function is only being called with rawValue.
https://github.com/IgniteUI/igniteui-angular/blob/73d7949c23b33cf3947a5b760df1ba8cdffd9589/projects/igniteui-angular/src/lib/services/exporter-common/base-export-service.ts#L471

Although the formatter function can receive a rowData, if I have a formatting function that uses rowData, it will not execute correctly at this point.
https://github.com/IgniteUI/igniteui-angular/blob/73d7949c23b33cf3947a5b760df1ba8cdffd9589/projects/igniteui-angular/src/lib/grids/common/grid.interface.ts#L137

Can we add the corresponding data in this call as well?

```diff
- let formattedValue = shouldApplyFormatter ? e.formatter(rawValue) : rawValue; 
+ let formattedValue = shouldApplyFormatter ? e.formatter(rawValue, record.data) : rawValue; 
```

**Comment by georgianastasov** [comment]
Hello @acemir,

In order to ensure that the described scenario is addressed correctly, I will need some additional information regarding your scenario. Could you please answer the following questions:

• Could you provide more detailed information about your requirements?
• For what purposes will you use the requested changes and what is the use case?
• Could you provide an isolated sample with steps to reproduce the described behavior?

This information is going to be highly appreciated and will help me in my further investigation to determine whether the given idea should be considered as a feature request.

**Comment by acemir** [comment]
Resolved #13234
